components/ProductCard/ProductCardBody.tsx

```diff
@@ -0,0 +1,16 @@
+import { chakra, Stack } from '@chakra-ui/react';
+import * as React from 'react';
+
+export interface ProductCardBodyProps {
```

Comment:
As far as I can tell, most of the time you can choose freely between types and interfaces as mentioned [here](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces).

Lately I've been trying to stick to use types since they support more use cases. But I have no strong opinions on this